package.json

```diff
@@ -18,7 +18,7 @@
   "homepage": "https://github.com/che-incubator/che-dashboard-next#readme",
   "engines": {
     "yarn": ">=1.20.0",
-    "node": ">=10.20.1"
+    "node": ">=12"
```

Comment:
@akurinnoy You are going to change the minimum requirement could you fix ```devfile.yaml``` too:
https://github.com/eclipse/che-dashboard/blob/main-react/devfile.yaml#L22
```docker.io/node:10.22.1``` -> ``` docker.io/node:12 ```

Reply:
rebased, updated.